src/GUI/EFCorePowerTools.Shared/Models/ModelingOptionsModel.cs

```diff
@@ -17,7 +17,8 @@ public class ModelingOptionsModel : INotifyPropertyChanged
         private bool _usePluralizer;
         private bool _useDatabaseNames;
         private string _ns;
-        private string _outputPath;
```

Comment:
Keep this to stay backwards compatible, and just add new optional path for DbContext